### PR TITLE
feat(#1330): Microsite header - open external urls in a new window

### PR DESCRIPTION
**Pull request by amar-ABgov**

### CODE REVIEW

File: libs/react-components/src/lib/microsite-header/microsite-header.tsx [hunk 1]
```diff
@@ -41,6 +47,8 @@ export const GoAMicrositeHeader: FC<HeaderProps> = ({
       feedbackurl={feedbackUrl}
       data-testid={testId}
       maxcontentwidth={maxContentWidth}
+      feedbackurlexternal={feedbackURLExternal}
```
Comment (chrisolsen) [inline libs/react-components/src/lib/microsite-header/microsite-header.tsx:50]: Keep the type case of `URL` consistent with `feedbackUrl`

File: libs/react-components/src/lib/microsite-header/microsite-header.tsx [hunk 2]
```diff
@@ -41,6 +47,8 @@ export const GoAMicrositeHeader: FC<HeaderProps> = ({
...
+      headerurlexternal={headerURLExternal}
```
Comment (chrisolsen) [inline libs/react-components/src/lib/microsite-header/microsite-header.tsx:51]: With this name it isn't obvious that it is a boolean variable. Maybe basing the name off the `<a target="_blank">` attribute would make things more clear. So using `feedbackUrlTarget="blank"` might make more sense.

File: libs/web-components/src/components/microsite-header/MicrositeHeader.html-data.json [hunk 3]
```diff
@@ -20,6 +20,38 @@
...
+        {
+          "name": "self",
+          "description": "target=_self"
+        },
+        {
+          "name": "blank",
+          "description": "target=_blank"
+        }
+      ],
+      "type": "boolean"
```
Comment (chrisolsen) [inline libs/web-components/src/components/microsite-header/MicrositeHeader.html-data.json:38]: This type need to be updated to match the `self | blank`

File: libs/web-components/src/components/microsite-header/MicrositeHeader.html-data.json [hunk 4]
```diff
@@ -20,6 +20,38 @@
       "default": "100%",
       "description": "Maximum width of the content area",
       "type": "string"
+    },
+    {
+      "name": "headerurltarget",
+      "default": "false",
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/microsite-header/MicrositeHeader.html-data.json:26]: Change to `default: self`

File: libs/web-components/src/components/microsite-header/MicrositeHeader.html-data.json [hunk 5]
```diff
@@ -20,6 +20,38 @@
...
+        {
+          "name": "blank",
+          "description": "target=_blank"
+        }
+      ],
+      "type": "boolean"
+    },
+    {
+      "name": "feedbackurltarget",
+      "default": "false",
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/microsite-header/MicrositeHeader.html-data.json:42]: Change to `default: self`

File: libs/docs/src/components/goa/microsite-header.stories.mdx [hunk 6]
```diff
@@ -34,6 +34,19 @@ import { Meta, Story } from "@storybook/addon-docs";
     defaultValue="100%"
     description="Maximum width of the content area"
   />
+  <Prop
+    name="feedbackUrlTarget"
+    type="self | blank"
+    defaultValue="self"
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/goa/microsite-header.stories.mdx:40]: After talking to @Spark450  and @twjeffery it was decided that these default values should actually be blank instead, so by default they open up in separate windows. Make sure this gets changed here, the `json` file, and the `svelte` file. Probably need to make some changes in tests as well.

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**